sklearn/ensemble/_forest.py

```diff
@@ -615,6 +615,13 @@ def feature_importances_(self):
     )
     @property
     def n_features_(self):
+        """Number of features when fitting the estimator.
+
+        Returns
+        -------
+        n_features_in_ : int
```

Comment:
The ending underscore causes sphinx to get confused apparently. We can try to not document with the name `n_features_in_` which is still compliant with numpydoc.

```suggestion
        int
```